ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -171,7 +171,7 @@ bracketChainSyncClient tracer ChainDbView { getIsInvalidBlock } varCandidates
 -- We must keep the candidate chain synchronised with the corresponding
 -- upstream chain. The upstream node's chain might roll forward or
 -- backwards, and they will inform us about this. When we get these
--- messages, we will replicate these actions on our candidate chain.
+-- messages, we will replicate these actions on our current chain.
```

Comment:
I think this is already an improvement; but maybe using "our fragment" is even slightly better as it is called `ourFrag` in the code and "current chain" often refers to our *selection* in other places (i.e. the method of the ChainDB to get the current selection is called `getCurrentChain`).

Reply:
Adopted in a38ce67.